hooks/finding-post-processing/hook/package.json

```diff
@@ -4,7 +4,6 @@
   "description": "secureCodeBox Finding Post Processing Hook",
   "main": "hook.js",
   "scripts": {
-    "build": "npx typescript hook.ts --sourceMap",
```

Comment:
Why is the script removed here, but updated in the cascading-scans hook?
@twwd

Reply:
The `find-post-processing` hook is written in plain JavaScript and thus requires no TypeScript transpilation. In turn, the `cascading-scans` hook is written in TypeScript like the `notification` hook (which was already updated to `tsc`).

Reply:
Haha ok good point.

Lost track what typescript and what is not :)